tests/README.md

```diff
@@ -1,8 +1,20 @@
+If you do not have the `fake-sys` directory it means you did not use the
+`--recursive` option when you git cloned the ev3dev-lang-python repo.
+Example:
+
+```
+$ git clone --recursive https://github.com/rhempel/ev3dev-lang-python.git
+```
+
 Commands used to copy the /sys/class node:
 
-```sh
-node=lego-sensor/sensor0
-mkdir -p ./${node}
-# Copy contents of special files, do not follow symlinks:
-cp -P --copy-contents -r /sys/class/${node}/* ./${node}/
+```
+$ node=lego-sensor/sensor0
+$ mkdir -p ./${node}
```

Comment:
I think this should be at the bottom, because it's rarely needed. The important command is the one  for running the tests.